### ISSUE TITLE
If you're not transmitting, you're not receiving either?

### ISSUE THREAD

**Issue by TheButterZone** [ge0rg/aprsdroid#224]
Around the times I transmit my position, I receive other stations. If I stop transmitting, clear logs & station list, I see the RX light on the radio light up, but nothing is decoded (while my SDR is feeding my decoding software just fine), no corresponding receive indicator moves on the app. Even if I switch from open squelch to closed squelch, no indication of the audio on/off. Using the Baofeng APRS-K2 cable, both leaving the volume knob where it previously allowed decoding, and also reducing/increasing volume from there.

`{
  "notify_led": false,
  "dgp_notify_ringtone": "content:\/\/media\/internal\/audio\/media\/10",
  "dgp_notify_led": false,
  "symbol": "\/p",
  "pos_notify_ringtone": "content:\/\/media\/internal\/audio\/media\/33",
  "activity": "hub",
  "distance": "0",
  "loc_source": "periodic",
  "sb.fastspeed": "100",
  "link": "bluetooth",
  "afsk.btsco": false,
  "service_running": false,
  "sb.turnmin": "10",
  "ssid": "",
  "afsk.hqdemod": true,
  "priv_spdbear": false,
  "frequency": "",
  "tcp.server": "euro.aprs2.net",
  "dgp_notify_vibr": false,
  "sb.fastrate": "60",
  "sb.turnslope": "240",
  "callsign": "mycallsign",
  "keepscreen": true,
  "tcp.sotimeout": "120",
  "afsk.prefix": "2000",
  "notify_vibr": false,
  "pos_notify_vibr": false,
  "firstrun": false,
  "afsk.output": "0",
  "digi_path": "WIDE1-1,WIDE2-1",
  "conn_log": true,
  "notify_ringtone": "content:\/\/media\/internal\/audio\/media\/11",
  "priv_altitude": false,
  "gps_activation": "med",
  "sb.slowspeed": "5",
  "priv_ambiguity": "3",
  "proto": "afsk",
  "pos_notify_led": false,
  "sb.slowrate": "1200",
  "sb.turntime": "15",
  "interval": "5",
  "tcp.filterdist": "50",
  "aprsis": "tcp",
  "status": "woof"
}
`

**Comment by TheButterZone** [comment]
I guess this is the way it's designed? Could there be a RX-only mode that doesn't send periodic positions?

**Comment by ge0rg** [comment]
You are right, this is an unfortunate effect of the original design of the app. I plan to allow an on-by-default RX only mode and to let the user trigger sending of beacons around the 2.0 release. This is actually long overdue :)

And yes, the options you have to receive only (which don't get your own position updated in APRSdroid, and thus also aren't perfect) are:
- disable GPS in the phone settings
- set APRSdroid to manual position without automatic posting
- APRS-IS only: set `-1` as your passcode. APRSdroid will still send packets, but they won't be accepted by the IS server

**Comment by arodland** [comment]
Related to this: I use aprsdroid with a TNC that I built myself which has its own GPS and does its own beaconing, but also speaks KISS. So I use aprsdroid in an "almost-receive-only" mode — it receives packets, displays them on the map, and can send and receive APRS messages.

As in your suggestion, I set APRSdroid to manual position with no automatic posting. I also have my TNC basically "local echo" all of the positions it beacons, so that APRSdroid sees them as received packets... and that actually causes APRSdroid to update my position, so that all of the distances and directions on the list view are correct! I'm not sure if that's intentional or an accidental side effect, but it's nice.

But what would be ideal for me would be a mode where APRSdroid doesn't attempt to beacon, but it still updates my location from the phone location. That way I could disable the "local echo" hack in my TNC, and APRSdroid would be constantly updating my location, rather than only when my box beacons.

**Comment by Luceoria** [comment]
>     * APRS-IS only: set `-1` as your passcode. APRSdroid will still send packets, but they won't be accepted by the IS server
Minus sign "-" seems to be not acceptable as a password in APRSdroid.

**Comment by mpannen1979** [comment]
> Related to this: I use aprsdroid with a TNC that I built myself which has its own GPS and does its own beaconing, but also speaks KISS. So I use aprsdroid in an "almost-receive-only" mode — it receives packets, displays them on the map, and can send and receive APRS messages.
> 
> As in your suggestion, I set APRSdroid to manual position with no automatic posting. I also have my TNC basically "local echo" all of the positions it beacons, so that APRSdroid sees them as received packets... and that actually causes APRSdroid to update my position, so that all of the distances and directions on the list view are correct! I'm not sure if that's intentional or an accidental side effect, but it's nice.
> 
> But what would be ideal for me would be a mode where APRSdroid doesn't attempt to beacon, but it still updates my location from the phone location. That way I could disable the "local echo" hack in my TNC, and APRSdroid would be constantly updating my location, rather than only when my box beacons.

I do this with Direwolf and rpi3 in hotspot configuration, with a GPS dongle.  Direwolf is configured as SMARTBEACONING,  with aprsdroid as the 'map head' connecting in TCPIP KISS.  I add a SENDTO=R0 in my direwolf.conf, and as long as MYCALL in direwolf matches the CALL-SSID in aprsdroid, I get my 'fresh' location in aprsdroid when direwolf beacons, the ability to send messages, and the smartbeacon from direwolf even if aprsdroid is not running.

**Comment by n3xus1** [comment]
mpannen1979 could you please show me your config with the SENDTO in direwolf and what you changed in APRSdroid. I'm doing the same as you but my connection is bluetooth to a fire tablet. I have most of it working except the distance to stations is wrong.
